sources/packages/backend/apps/api/src/route-controllers/education-program/education-program.aest.controller.ts

```diff
@@ -123,4 +128,29 @@ export class EducationProgramAESTController extends BaseController {
       userToken.userId,
     );
   }
+
+  /**
+   * Allows a program to be deactivated.
+   * @param programId program to be deactivated.
+   * @param payload information to support the deactivation.
+   */
+  @ApiNotFoundResponse({
+    description: "Not able to find the education program.",
+  })
+  @ApiUnprocessableEntityResponse({
+    description: "The education program is already set as requested.",
+  })
+  @Roles(Role.InstitutionDeactivateEducationProgram)
```

Comment:
Do we want this rolename to have the `institution` prefix in it? I see that both the institution and the ministry have the capability to deactivate the program.

Reply:
Roles are Ministry specific and they are prefixed with `institution` or `student` usually.
Please see below the other institution roles used by Ministry.
institution-edit-profile
institution-approve-decline-program
institution-approve-decline-offering
institution-edit-location-details
institution-add-new-user
institution-edit-user
institution-enable-disable-user
institution-add-restriction
institution-resolve-restriction
institution-create-note
institution-approve-decline-designation
institution-approve-decline-offering-changes
institution-approve-decline-application-offering-change-request
institution-deactivate-education-program